### ISSUE TITLE
v0.95.0 is missing otel-contrib rpm files

### ISSUE THREAD

**Issue by josephmask** [open-telemetry/opentelemetry-collector-releases#487]
We have recently begun using the latest otel-contrib rpm files until this week when v0.95.0 was published and we noticed it did not include otel-contrib rpms in the release.

We expected to be able to download https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.95.0/otelcol-contrib_0.95.0_linux_amd64.rpm, but it does not exist.

Was this intentional or a defect?

Thanks.

**Comment by TylerHelmuth** [comment]
This is definitely a bug, most likely a result of https://github.com/open-telemetry/opentelemetry-collector-releases/pull/477

**Comment by TylerHelmuth** [comment]
Yes it is because I didn't add a `builds` section to the `nfpm` config.  I mistakenly thought the `id` in the section was what I wanted.  

We may be doing a patch release this week for some other stuff.  I'll try to get this fixed up before then.

**Comment by TylerHelmuth** [comment]
Hmm maybe not, because `builds` was already there.  I'll need to keep investigating.

**Comment by TylerHelmuth** [comment]
It worked here for Core: https://github.com/open-telemetry/opentelemetry-collector-releases/actions/runs/7992182647/job/21825323421

But here it failed contrib: https://github.com/open-telemetry/opentelemetry-collector-releases/actions/runs/7992182657/job/21825106699

**Comment by TylerHelmuth** [comment]
This will be fixed by https://github.com/open-telemetry/opentelemetry-collector-releases/pull/485.

You can see in https://github.com/open-telemetry/opentelemetry-collector-releases/actions/runs/8012970345/job/21889235651?pr=485 and https://github.com/open-telemetry/opentelemetry-collector-releases/actions/runs/8012970348/job/21889235648?pr=485 that for both contrib and core the linux packages are now correctly built.
